src/cpp/include/openvino/genai/scheduler_config.hpp

```diff
@@ -30,5 +30,8 @@ struct SchedulerConfig {
 
     // max number of scheduled sequences (you can think of it as "max batch size")
     std::size_t max_num_seqs = 256;
+
+    // enable caching of KV-blocks and reusage in next generations.
+    bool enable_prefix_caching = false;
```

Comment:
Why is it disabled by default? Why would anyone want to configure that in the first place?

Reply:
User might need to configure it because enabling prefix caching uses more RAM. All previously calculated KV-cache is kept in memory for future usages. KV-caches can be rewritten if we reached cache limit, but blocks are not actually released.

In case of disabling prefix caching we use only KV-cache required for batch calculation and when sequence is finished cache is released. So I think this parameter might be useful.

Regarding default value, we can set to true, it's not required to be false.

Reply:
Please add this explanation to docstrings.

Who are the beneficiaries of this feature? I hope this may help to pick the default value.

Reply:
One approach to that question is when chat becomes efficient in CB, enable_prefix_caching must become true by default.

Reply:
Updated docstring. 
Regarding default value, I suppose we need to set it to true for chat scenario only. I don't think, that it is needed to be enabled by default.